compiler/x/codegen/OMRTreeEvaluator.cpp

```diff
@@ -6117,13 +6117,79 @@ OMR::X86::TreeEvaluator::iexpandbitsEvaluator(TR::Node *node, TR::CodeGenerator
 TR::Register*
 OMR::X86::TreeEvaluator::mAnyTrueEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    {
-   return TR::TreeEvaluator::unImpOpEvaluator(node, cg);
+   TR::Node *maskNode = node->getFirstChild();
+   TR::Register *maskReg = cg->evaluate(maskNode);
+   TR::Register *resultReg = cg->allocateRegister();
+
+   if (maskReg->getKind() == TR_VMR && cg->comp()->target().cpu.supportsFeature(OMR_FEATURE_X86_AVX512BW))
+      {
+      TR_ASSERT_FATAL(cg->comp()->target().cpu.supportsFeature(OMR_FEATURE_X86_AVX512F), "Mask registers require AVX-512");
```

Comment:
Cache `cg->comp()->target()` in a local to improve readability and (minimally) compile performance.